easybuild/easyconfigs/t/typing-extensions/typing-extensions-3.7.4.3-fosscuda-2020b.eb

```diff
@@ -0,0 +1,21 @@
+easyblock = 'PythonPackage'
```

Comment:
There already is a typing-extensions-3.7.4.3-GCCcore-10.2.0.eb in the EC repo, so we don't need this one

Reply:
Thanks. I've deleted it.